### PR TITLE
Remove migrated integration tests from CI altogether

### PR DESCRIPTION
**Pull request by QuiteStochastic**
there's one last migrated integration test in the code but it's not running in CI anyways.  the last test will have it's logic ported over to the new framework eventually, but until in any case we don't need migrated integration tests in CI anymore

### CODE REVIEW

**Comment by QuiteStochastic** [comment]
!ci-build-me

**Comment by QuiteStochastic** [comment]
!ci-build-me

**Comment by QuiteStochastic** [comment]
!ci-build-me

**Comment by QuiteStochastic** [comment]
!ci-build-me

**Comment by QuiteStochastic** [comment]
!ci-build-me

**Comment by QuiteStochastic** [comment]
!ci-build-me

File: src/app/cli/src/cli_entrypoint/mina_cli_entrypoint.ml [hunk 1]
```diff
@@ -1697,9 +1697,9 @@ let mina_commands logger =
   let group =
     List.map
       ~f:(fun (module T) -> (T.name, T.command))
-      ( [ (module Coda_shared_state_test)
-        ; (module Coda_transitive_peers_test)
-        ; (module Coda_change_snark_worker_test)
+      ( [ (* (module Coda_shared_state_test)
```
Comment (psteckler) [inline src/app/cli/src/cli_entrypoint/mina_cli_entrypoint.ml:1700]: would be good to include in the comment the status of these tests, why they're being disabled

presumably, you're leaving the comment for the possibility of reviving these tests

**Review by mrmr1993 — APPROVED** [review]

File: src/app/cli/src/cli_entrypoint/mina_cli_entrypoint.ml (hunk 1 above)
Comment (QuiteStochastic) [reply src/app/cli/src/cli_entrypoint/mina_cli_entrypoint.ml:1700]: these will all go away soon

**Comment by QuiteStochastic** [comment]
!ci-build-me

**Review by mrmr1993 — APPROVED** [review]

**Review by stevenplatt — APPROVED** [review]
Approved

**Pull request merged**